swift/Workflow/Sources/SubtreeManager.swift

```diff
@@ -190,6 +210,46 @@ extension WorkflowNode.SubtreeManager {
 
 }
 
+
+extension WorkflowNode.SubtreeManager {
+    fileprivate final class SinkContext {
```

Comment:
We use the `context` term in a bunch of places, I hope we can find a more descriptive word here. No brilliant suggestions from me at the moment, however.

Reply:
+1. It's a bad name for this (since we already have a bunch of other things with this name). I'll come up with another one.

Reply:
Ended up with `ActionPipe` as it's the pipe on the sink that actions flow down.

I'm still not 100% happy with it, but it's at least not a collision with "context".

`ActionDrain`? `ActionHandler`?

Reply:
😂 I almost suggested pipe.